src/main/java/core/basesyntax/service/Parser.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax.service;
+
+import java.util.List;
+
+public interface Parser {
+
+    abstract List<String> parseLine(String line);
```

Comment:
If you use empty line after declaration of interface so use it for every that looks in same style